lib/tool_belt/koji_tag.rb

```diff
@@ -21,14 +21,13 @@ def process
         tools.ensure_build_target(name, build_target)
 
         if build_package_group_source_tag
-          tools.ensure_package_group(build_package_group_source_tag, build_target, 'build')
-          tools.ensure_package_group(build_package_group_source_tag, build_target, 'srpm-build')
+          tools.ensure_package_group(build_target, 'build', :source_tag => build_package_group_source_tag)
+          tools.ensure_package_group(build_target, 'srpm-build', :source_tag => build_package_group_source_tag)
         end
 
         if build_groups
           build_groups.each do |group, packages|
-            tools.create_package_group(build_target, group)
-            tools.add_group_packages(build_target, group, packages)
+            tools.ensure_package_group(build_target, group, packages: packages)
```

Comment:
above you write `:source_tag => build_package_group_source_tag` but here you use `packages: packages`, while this is equivalent in modern ruby, I don't think we should mix styles here?

Reply:
That's fair -- updated